sensing/image_diagnostics/README.md

```diff
@@ -0,0 +1,41 @@
+# image_diagnostics
+
+## Purpose
+
+The `image_diagnostics` is a node that check the status of the input raw image.
+
+## Inner-workings / Algorithms
+
```

Comment:
[IMO] Could you explain some description about how this node works?

Reply:
Also, please describe more detailed usage of this in Purpose. (When this can be used? How?)

Reply:
Thanks I will revise it